### DISCUSSION TITLE
How to Change the Blue Header Color of Notifications in My Application?

### DISCUSSION THREAD

**Discussion by maguro1kan** [AvaloniaUI/Avalonia#16882] [Q&A] [answered]
I would like to change the color of the top blue part of the notification in the bottom right corner of the attached image. I want to change it from blue to transparent or match it with the background color of the notification.

Could you please guide me on how to achieve this?

![image](https://github.com/user-attachments/assets/b1f76609-0fd8-4304-adc1-fd5d5a683082)

Thanks

**Comment by Coloryr** [comment, marked as answer]
https://github.com/Coloryr/ColorMC/blob/master/src/ColorMC.Gui/Style/WindowNotificationManager.axaml
